### PR TITLE
Create an API for the tasks

### PR DESCRIPTION
**Pull request by ortoLover**
We already have done the API for multiple values. We now need to concentrate ourselves in the task API, so that we can implement the map later, and the list of tasks. The basic API should contain:

- [x] An updated version of the tasks
- [x] Get all tasks
- [x] Get a task
- [x] Edit a task
- [x] Add a task
- [x] Delete a task

### CODE REVIEW

File: app/src/main/java/com/github/se/assocify/model/entities/Task.kt [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
-    val name: String = "testName",
+    val title: String = "testName",
     val description: String = "description",
     val isCompleted: Boolean = false,
-    val startTime: String = LocalDate.now().format(DateTimeFormatter.ofPattern("dd/MM/yyyy")),
+    val startTime: LocalDate = LocalDate.now(),
     val peopleNeeded: Int = 0,
     val category: String = "Committee",
-    val location: String = "Here"
+    val location: String = "Here",
```
Comment (sarahbadr17) [inline app/src/main/java/com/github/se/assocify/model/entities/Task.kt:26]: I don't think location should be a string. It should be coordinates no?

**Review by sarahbadr17 — COMMENTED** [review]
LGTM!

File: app/src/main/java/com/github/se/assocify/model/entities/Task.kt (hunk 1 above)
Comment (ortoLover) [reply app/src/main/java/com/github/se/assocify/model/entities/Task.kt:26]: We still haven't decided how to store the location. We have thought of doing a class location where we store location, but it's still in discussion. I thought about putting a string temporarly, but we can modify it if you want

Comment (sarahbadr17) [reply app/src/main/java/com/github/se/assocify/model/entities/Task.kt:26]: It's fine for now just add an issue in the backlog to modify this !

**Review by sarahbadr17 — APPROVED** [review]
LGTM! good job

**Pull request merged**